src/utils.js

```diff
@@ -1,6 +1,29 @@
 import _ from 'lodash';
 
 export const utils = {
+  /**
```

Comment:
just make a simple tryParse util instead, otherwise you are parsing twice
`
tryParse = data => {
  try {
    return JSON.parse(data);
  } catch (err) {
    return data;
  }
}
`